### PR TITLE
Groups grid UI enhancements

### PR DESCRIPTION
**Pull request by comzeradd**
This a screenshot of the new Goups list UI, following what we already have on Group details.

screen shot 2016-04-05 at 12 39 46

### CODE REVIEW

**Comment by ppapadeas** [comment]
r+ for UI.

**Comment by comzeradd** [comment]
@akatsoulas r?

**Comment by comzeradd** [comment]
Rebased to master

File: mozillians/jinja2/groups/group.html [hunk 1]
```diff
@@ -52,6 +52,29 @@ <h1 id="group_name">
       </a>
     {% endif %}
 
+    {% if show_join_button %}
```
Comment (akatsoulas) [inline mozillians/jinja2/groups/group.html]: screen shot 2016-12-29 at 15 24 12
The buttons don't look very good in the mobile view (nexus 5 simulated device). Maybe we should move them below the 'skills in common' section. Thoughts?

**Comment by comzeradd** [comment]
On your schreenshot that group lacks description, making the buttons seem to close to the rest of text. I amended the css a bit to add some space even when group description is not there.

**Pull request closed without merging**